packages/i18n/I18nDecorator/getI18nClasses.js

```diff
@@ -45,7 +45,7 @@ function getClassesForLocale (li, options) {
 		})
 	], function (classes) {
 		const scriptName = li.getScript();
-		if (scriptName !== 'Latn' && scriptName !== 'Cyrl' && scriptName !== 'Grek') {
+		if (scriptName !== 'Latn' && scriptName !== 'Cyrl' && scriptName !== 'Grek' && scriptName !== 'Kore') {
```

Comment:
I'm not sure that this condition is fine to all products because the font in Auto doesn't have italic font in it. There are 2 options.
1) Merge first, then update the auto font or create an additional PR for auto
2) Add more condition to check if it is on TV from enact/webOS